routers/index.js

```diff
@@ -655,7 +655,7 @@ var mime = 'html';
     if (!error && response1.statusCode == 200) {
       response.writeHead(200, {'Content-Type': 'text/'+mime});
       response.write('<h1>Welcome to Ninzas\'s SSRF demo.</h1>\n\n');
-      response.write('<h2>I am an application. I want to be useful, so I requested: <font color="red">url</font> for you\n</h2><br><br>\n\n\n');
+      response.write('<h2>I am an application. I want to be useful, so I requested: <font color="red">'+url+'</font> for you\n</h2><br><br>\n\n\n');
```

Comment:
#### Automated PR Comment From coverity
<!-- 3003290b1c5dc116c1514fc3889a755d -->
<!-- Comment managed by Git Commenter, do not modify! -->
### Coverity Issue - Cross-site scripting
_High CWE-79_
Calling "response.write" with the tainted value in ""<h2>I am an application. I want to be useful, so I requested: <font color=\"red\">" + url + "</font> for you\n</h2><br><br>\n\n\n"". The untrusted data reaches a sink that may allow an attacker to control part of the response.

A user can execute arbitrary JavaScript on a web page viewed or accessed by another user, potentially allowing session hijacking, disclosing sensitive data in the DOM, or viewing of keyboard and mouse events.
### How to fix
Escape non-constant data appropriately before concatenating it into HTML. The specific sequence of escapers necessary to make data safe depends on its syntactic position in the HTML. Allowing only safe characters (whitelisting) sometimes suffices to avoid XSS vulnerabilities, but only the strictest whitelists prevent all attacks.